src-ghc/Pact/Main.hs

```diff
@@ -31,7 +31,7 @@ import Text.Trifecta as TF hiding (err)
 import System.IO
 import System.Exit hiding (die)
 import qualified Options.Applicative as O
-import Data.Monoid
+import Data.Monoid ()
```

Comment:
Is this and related changes required for 8.4? Does it warn on instances-only imports or something?

Reply:
Yes, these changes are required for 8.4, as `(<>)` is now provided by `Semigroup`.

Reply:
That doesn't answer the question why Monoid would become an instances-only import here.

Reply:
Ah, you would like it removed where it isn't required. I will update this.

Reply:
Okay, updated.